docs/standard/microservices-architecture/docker-application-development-process/docker-app-development-workflow.md

```diff
@@ -1,216 +1,339 @@
 
 **Figure 5-4**. Enabling Docker support in an existing Visual Studio 2017 project
 
-This action adds a *Dockerfile* to the project with the required configuration, and is only available on .NET Core web app projects.
-
-To add a *docker-compose.yml* file for the whole solution, right-click on the project in **Solution Explorer** and select **Add** > **Container Orchestrator Support**, as shown in Figure 5-5.
-
-![Add container orchestrator support menu option in Visual Studio](./media/add-container-orchestrator-support.png)
-
-**Figure 5-5**. Adding container orchestrator support to an existing project in Visual Studio 2017.
+This action adds a *Dockerfile* to the project with the required configuration, and is only available on ASP.NET Core projects.
 
-In the following sections, we describe the information that goes into each of those files. Visual Studio can do this work for you, but it is useful to understand what goes into a Dockerfile.
+In a similar fashion, Visual Studio can also add a docker-compose.yml file for the whole solution with the option **Add > Container Orchestrator Support**. In step 4 we'll explore this option with greater detail.
 
-### Option A: Creating a project using an existing official .NET Docker image
+### Using an existing official .NET Docker image
 
-You usually build a custom image for your container on top of a base image you can get from an official repository at the [Docker Hub](https://hub.docker.com/) registry. That is precisely what happens under the covers when you enable Docker support in Visual Studio. The Dockerfile uses an existing aspnetcore image.
+You usually build a custom image for your container on top of a base image you get from an official repository like the [Docker Hub](https://hub.docker.com/) registry. That is precisely what happens under the covers when you enable Docker support in Visual Studio. Your Dockerfile will use an existing aspnetcore image.
 
-Earlier we explained which Docker images and repos you can use, depending on the framework and OS you have chosen. For instance, if you want to use ASP.NET Core (Linux or Windows), the image to use is microsoft/aspnetcore:2.0. Therefore, you just need to specify what base Docker image you will use for your container. You do that by adding FROM microsoft/aspnetcore:2.0 to your Dockerfile. This is automatically performed by Visual Studio, but if you were to update the version, you update this value.
+Earlier we explained which Docker images and repos you can use, depending on the framework and OS you have chosen. For instance, if you want to use ASP.NET Core (Linux or Windows), the image to use is microsoft/dotnet:2.1-aspnetcore-runtime. Therefore, you just need to specify what base Docker image you will use for your container. You do that by adding FROM microsoft/dotnet:2.1-aspnetcore-runtime to your Dockerfile. This will be automatically performed by Visual Studio, but if you were to update the version, you update this value.
```

Comment:
these image names throughout the text need to be involved in backticks to lock them for localization like in line 109